app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionTabs/makeDecisionTabs.jsx

```diff
@@ -104,8 +103,12 @@ export const MakeDecisionTabs = ({
   };
 
   return (
-    <div className={cx('make-decision-tabs', { 'bulk-operation': isBulkOperation })}>
-      {(!isBulkOperation || isMLSuggestionsAvailable) && (
+    <div
+      className={cx('make-decision-tabs', {
+        'bulk-operation': !isMLSuggestionsAvailable,
```

Comment:
Now this component doesn't know anything about bulk operation. May be better to rename the `className` here?